app/controllers/api/v1/chat_rooms_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class ChatRoomsController < ApiController
+      def index
+        @chat_rooms = current_user.chat_rooms
+      end
+
+      def show
+        @chat_room = current_user.chat_rooms.includes(:messages).find_by!(id: params[:id])
```

Comment:
There is no need to pre include messages when there is only one chat_room.
Also, just use `find` when searching by `id`